packages/frontend/src/hooks/useExplorerRoute.ts

```diff
@@ -54,17 +54,14 @@ export const useExplorerRoute = () => {
 
     // Update url params based on pristine state
     useEffect(() => {
-        if (queryResultsData?.metricQuery) {
-            history.replace(
-                getExplorerUrlFromCreateSavedChartVersion(
-                    pathParams.projectUuid,
-                    {
-                        ...unsavedChartVersion,
-                        metricQuery: queryResultsData.metricQuery,
-                    },
-                ),
-            );
-        }
```

Comment:
Reseting query means reseting `explorer state` and then `query results`, these state changes causes one render for each one.
1º render: `unsavedChartVersion` as `default state` and `queryResults` with values
2º render: `unsavedChartVersion` as `default state` and `queryResults` as `undefined`

This verification skipped the url update on the (2), so even though there is nothing selected, `metricQuery` on url still had previous values.
As I thought this verification a bit strange and I didn't find any difference or bug after removing it, I chose this solution, if it breaks something I'd change the approach without any problem

Reply:
The only problem I seen is that the url params aren't cleared when the user navigates back to the explores page, to pick a new table.

Perhaps we need to keep a condition `if(!unsavedChartVersion.tableName)`

Reply:
I noticed that when the user navigates back, the **url params is present but with the default** `unsavedChartVersion` state, so it's being cleared 🤔 

This happens because without the condition, the url params is filled with the default `unsavedChartVersion` state as soon as the user enters `explorer page`

Do you think that the url params being present with default `unsavedChartVersion` is a problem? I saw that `run query` and `save chart` button are still disabled in this case 🤔